apps/users/models.py

```
@@ -53,3 +45,25 @@ class OrganizationUser(models.Model):
     user = models.ForeignKey(User, on_delete=models.CASCADE)
     organization = models.ForeignKey("organizations.Organization", on_delete=models.CASCADE)
     is_owner = models.BooleanField(default=False)
+
+
+class Roles(models.TextChoices):
+    HUB_ADMIN = "hub_admin", "Hub Admin"
+    ORG_ADMIN = "org_admin", "Org Admin"
+    RP_ADMIN = "rp_admin", "RP Admin"
+
+
+class RoleAssignment(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    role_type = models.CharField(max_length=max(len(x) for x in Roles.values), choices=Roles.choices)
```

Rusty on django, help me out. I'm queasy about dynamically generated model field parameter

Will the db migration generator tool (forget name, South?) detect and create migration to change size if we add a longer role?

[Also with Postgres having tight max_length doesn't get you anything (re storage) and I think it silently truncates.  you do get validation on the django side. I guess I'm asking why not just max_length 20 or 40]

---

I believe the Django's migration capabilities would catch the change, but I agree, @njharman I'm not a fan of the dynamic generation of the role type. I'm going to update.